### ISSUE TITLE
FR: Use skaffold for dev / building

### ISSUE THREAD

**Issue by blaggacao** [openintegrationhub/openintegrationhub#1006]
Implement https://skaffold.dev

Given the well defined repository structure it's rather trivial - at first sight. I might do a PR in the coming days.

**Comment by blaggacao** [comment]
I'm working on an implementation at https://github.com/xoe-labs/openintegrationhub/pull/2

Please have a loop to see what it's all about.

**Comment by blaggacao** [comment]
Currently, this issue is out of scope for immediate review. After #1074 is resolved I might re-postulate the issue with a more complete view how things would be able to fit together.
